pkg/gatherers/conditional/container_log.schema.json

```
@@ -34,7 +34,7 @@
             "uniqueItems": false,
             "items": {
                 "type": "string",
-                "description": "Regular expression used to filter the container log"
+                "description": "Regular expression used to filter the container log. The Go's Match (https://pkg.go.dev/regexp#Regexp.Match) function is used, so if a log line has a match, it is included in the collected data."
```

The wording is developer-centric a bit more than I would expect in this context. :) More importantly, the causality between using the Go's Match function and the rest of the sentence doesn't feel right.

How about this?

```suggestion
                "description": "Regular expression used to filter the container log. The collected data will include all lines matching the regular expression (as implemented by https://pkg.go.dev/regexp#Regexp.Match)."
```

---

OK :) Thanks